### PR TITLE
Fixed links🔗

### PR DESCRIPTION
**Pull request by himanshu007-creator**
## Description
This Pull Request solves the links issues. All references of <code>https://github.com/cncf/cnf-conformance</code> are replaced to <code>https://github.com/cncf/cnf-conformance</code>
In a file [USAGE.md], all references are changed to relative links

## Issues:
Refs: #724 

## How has this been tested:
 - [ ] Covered by existing integration testing
 - [ ] Added integration testing to cover
 - [ ] Verified all A/C passes
     * [ ] develop
     * [ ] master
     * [ ] tag/other branch
 - [ ] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [ ] Kind cluster
 - [ ] Have not tested

## Types of changes:
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [x] Documentation update

## Checklist:
**Documentation**
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] No updates required.

**Code Review**
- [ ] Does the test handle fatal exceptions, ie. rescue block

**Issue**
- [x] Tasks in issue are checked off

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -60,4 +61,4 @@ Also setting the verbose option for many tasks will add extra output to help wit
 crystal src/cnf-conformance.cr test_name verbose
 ```
 
-Check [usage documentation](https://github.com/cncf/cnf-conformance/blob/master/USAGE.md) for more info about invoking commands and loggin
+Check [usage documentation](https://github.com/cncf/cnf-testsuite/blob/master/USAGE.md) for more info about invoking commands and loggin
```
Comment (taylor) [inline .github/ISSUE_TEMPLATE/bug-report.md:64]: Please use a relative path.

File: .github/ISSUE_TEMPLATE/new-example-cnf.md [hunk 2]
```diff
@@ -3,65 +3,72 @@ name: New Example CNF
...
 **Documentation tasks:**
-- [ ] Update [Test Categories md](https://github.com/cncf/cnf-conformance/blob/master/TEST-CATEGORIES.md) if needed 
-- [ ] Update [Pseudo Code md](https://github.com/cncf/cnf-conformance/blob/master/PSEUDO-CODE.md) if needed
-- [ ] Update [USAGE md](https://github.com/cncf/cnf-conformance/blob/master/USAGE.md) if needed 
-- [ ] Update [installation instructions](https://github.com/cncf/cnf-conformance#installation)  if needed 
 
+- [ ] Update [Test Categories md](TEST-CATEGORIES.md) if needed
+- [ ] Update [Pseudo Code md](PSEUDO-CODE.md) if needed
+- [ ] Update [USAGE md](USAGE.md) if needed
+- [ ] Update [installation instructions](https://github.com/cncf/cnf-testsuite/#installation) if needed
```
Comment (taylor) [inline .github/ISSUE_TEMPLATE/new-example-cnf.md:50]:
```suggestion
- [ ] Update [installation instructions](INSTALL.md) if needed
```

File: .github/ISSUE_TEMPLATE/new-platform-test.md [hunk 3]
```diff
@@ -3,68 +3,76 @@ name: New Platform Test
...
 **Documentation tasks:**
-- [ ] Update [Test Categories md](https://github.com/cncf/cnf-conformance/blob/master/TEST-CATEGORIES.md) if needed 
-- [ ] Update [Pseudo Code md](https://github.com/cncf/cnf-conformance/blob/master/PSEUDO-CODE.md) if needed
-- [ ] Update [USAGE md](https://github.com/cncf/cnf-conformance/blob/master/USAGE.md) if needed 
-- [ ] Update [installation instructions](https://github.com/cncf/cnf-conformance#installation)  if needed 
 
+- [ ] Update [Test Categories md](TEST-CATEGORIES.md) if needed
+- [ ] Update [Pseudo Code md](PSEUDO-CODE.md) if needed
+- [ ] Update [USAGE md](USAGE.md) if needed
+- [ ] Update [installation instructions](https://github.com/cncf/cnf-testsuite/#installation) if needed
```
Comment (taylor) [inline .github/ISSUE_TEMPLATE/new-platform-test.md:54]:
```suggestion
- [ ] Update [installation instructions](INSTALL.md) if needed
```

File: .github/ISSUE_TEMPLATE/new-workload-test.md [hunk 4]
```diff
@@ -3,71 +3,80 @@ name: New Workload Test
...
 **Documentation tasks:**
-- [ ] Update [Test Categories md](https://github.com/cncf/cnf-conformance/blob/master/TEST-CATEGORIES.md) if needed 
-- [ ] Update [Pseudo Code md](https://github.com/cncf/cnf-conformance/blob/master/PSEUDO-CODE.md) if needed
-- [ ] Update [USAGE md](https://github.com/cncf/cnf-conformance/blob/master/USAGE.md) if needed 
-- [ ] Update [installation instructions](https://github.com/cncf/cnf-conformance#installation)  if needed 
 
+- [ ] Update [Test Categories md](TEST-CATEGORIES.md) if needed
+- [ ] Update [Pseudo Code md](PSEUDO-CODE.md) if needed
+- [ ] Update [USAGE md](USAGE.md) if needed
+- [ ] Update [installation instructions](https://github.com/cncf/cnf-testsuite/#installation) if needed
```
Comment (taylor) [inline .github/ISSUE_TEMPLATE/new-workload-test.md:58]:
```suggestion
- [ ] Update [installation instructions](INSTALL.md) if needed
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,36 +1,36 @@
...
+The CNF Test Suite is [Apache 2.0 licensed](LICENSE) and accepts contributions via GitHub pull requests. Please read the following guidelines carefully to make it easier to get your contribution accepted.
+
+## Support Channels:
 
-Support Channels:
----
 Support channels include:
-- [Issues](https://github.com/cncf/cnf-conformance/issues)
+
+- [Issues](https://github.com/cncf/cnf-testsuite/issues)
```
Comment (taylor) [inline CONTRIBUTING.md:11]:
```suggestion
- [Issues](issues)
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,36 +1,36 @@
...
 - Slack:
-    - [#cnf-conformance-dev](https://cloud-native.slack.com/archives/C014TNCEX8R) 
-    - [#cnf-conformance](https://cloud-native.slack.com/archives/CV69TQW7Q)
+  - [#cnf-conformance-dev](https://cloud-native.slack.com/archives/C014TNCEX8R)
+  - [#cnf-conformance](https://cloud-native.slack.com/archives/CV69TQW7Q)
 
-Before starting work on a major feature, please reach out to us via [GitHub Issues](https://github.com/cncf/cnf-conformance/issues) or Slack. We will make sure no one else is already working on it and ask you to open a [GitHub issue](https://github.com/cncf/cnf-conformance/issues/new/choose).
-- Small patches and bug fixes don't need prior communication.
+Before starting work on a major feature, please reach out to us via [GitHub Issues](https://github.com/cncf/cnf-testsuite/issues) or Slack. We will make sure no one else is already working on it and ask you to open a [GitHub issue](https://github.com/cncf/cnf-testsuite//issues/new/choose).
```
Comment (taylor) [inline CONTRIBUTING.md:16]:
```suggestion
Before starting work on a major feature, please reach out to us via [GitHub Issues](issues) or Slack. We will make sure no one else is already working on it and ask you to open a [GitHub issue](issues/new/choose).
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,36 +1,36 @@
...
----
-Issues are used as the primary method for tracking items in the CNF Test Suite initiative. Please self-assign an issue to yourself when you start to work on it so we don't duplicate work :) 
+## Issues
 
-- [Issues](https://github.com/cncf/cnf-conformance/issues)
-    - [Good first issues](https://github.com/cncf/cnf-conformance/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
+Issues are used as the primary method for tracking items in the CNF Test Suite initiative. Please self-assign an issue to yourself when you start to work on it so we don't duplicate work :)
 
+- [Issues](https://github.com/cncf/cnf-testsuite//issues)
+  - [Good first issues](https://github.com/cncf/cnf-testsuite/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
```
Comment (taylor) [inline CONTRIBUTING.md:25]:
```suggestion
  - [Good first issues](issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
```

**Review by taylor — CHANGES_REQUESTED** [review]
@himanshu007-creator please update the links which were changed in this PR to be relative links.  I've shown a few suggested, there are many more though in this PR.

File: tools/ephemeral_env/README.md [hunk 8]
```diff
@@ -1,60 +1,80 @@
...
+## Ephemeral Development Environment
 
-The enviroment is designed to be seamless and invisible to the developer.  It takes the regular crystal commands and passes them into the docker environment.  It also has intelligent integration with the cnf-conformance binary.
+The Ephemeral Developement Environment is a developer environment that allows the user to contribute to the cnf-conformance project without having crystall lang installed on the developer's host machine. The enviroment consists of a docker container which has the correct version of crystal lang for cnf-conformance development installed.
+
+The enviroment is designed to be seamless and invisible to the developer. It takes the regular crystal commands and passes them into the docker environment. It also has intelligent integration with the cnf-conformance binary.
+
+## Quickstart
+
+Prereqs: https://github.com/cncf/cnf-testsuite/blob/master/KIND-INSTALL.md
```
Comment (agentpoyo) [inline tools/ephemeral_env/README.md:9]:
```suggestion
Prereqs: https://github.com/cncf/cnf-testsuite/blob/main/KIND-INSTALL.md
```

**Comment by agentpoyo** [comment]
> @himanshu007-creator please update the links which were changed in this PR to be relative links. I've shown a few suggested, there are many more though in this PR.

The relative paths won't work in the issue templates, need the full path URL since the issues don't sit in the same root directory. I don't see the templates changing again unless we change the default branch and or the doc they're linking to changes (name or header link).

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (himanshu007-creator) [reply .github/ISSUE_TEMPLATE/bug-report.md:64]: wont work in ISSUE TEMPLATE 
reference:-https://github.com/cncf/cnf-testsuite/pull/742#issuecomment-827820452

File: .github/ISSUE_TEMPLATE/new-example-cnf.md (hunk 2 above)
Comment (himanshu007-creator) [reply .github/ISSUE_TEMPLATE/new-example-cnf.md:50]: wont work in ISSUE TEMPLATE 
reference:-https://github.com/cncf/cnf-testsuite/pull/742#issuecomment-827820452

File: CONTRIBUTING.md (hunk 5 above)
Comment (himanshu007-creator) [reply CONTRIBUTING.md:11]: done

File: CONTRIBUTING.md (hunk 6 above)
Comment (himanshu007-creator) [reply CONTRIBUTING.md:16]: done

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (himanshu007-creator) [reply .github/ISSUE_TEMPLATE/bug-report.md:64]: done

File: .github/ISSUE_TEMPLATE/new-example-cnf.md [hunk 9]
```diff
@@ -3,65 +3,72 @@ name: New Example CNF
...
+
+- [ ]
 
 **Documentation tasks:**
-- [ ] Update [Test Categories md](https://github.com/cncf/cnf-conformance/blob/master/TEST-CATEGORIES.md) if needed 
-- [ ] Update [Pseudo Code md](https://github.com/cncf/cnf-conformance/blob/master/PSEUDO-CODE.md) if needed
-- [ ] Update [USAGE md](https://github.com/cncf/cnf-conformance/blob/master/USAGE.md) if needed 
-- [ ] Update [installation instructions](https://github.com/cncf/cnf-conformance#installation)  if needed 
 
+- [ ] Update [Test Categories md](TEST-CATEGORIES.md) if needed
```
[the remark this answers is not in the document]
Comment (himanshu007-creator) [reply .github/ISSUE_TEMPLATE/new-example-cnf.md:47]: done

File: .github/ISSUE_TEMPLATE/new-example-cnf.md [hunk 10]
```diff
@@ -3,65 +3,72 @@ name: New Example CNF
...
+- [ ] Update [Pseudo Code md](PSEUDO-CODE.md) if needed
```
[the remark this answers is not in the document]
Comment (himanshu007-creator) [reply .github/ISSUE_TEMPLATE/new-example-cnf.md:48]: done

File: .github/ISSUE_TEMPLATE/new-example-cnf.md [hunk 11]
```diff
@@ -3,65 +3,72 @@ name: New Example CNF
...
+- [ ] Update [USAGE md](USAGE.md) if needed
```
[the remark this answers is not in the document]
Comment (himanshu007-creator) [reply .github/ISSUE_TEMPLATE/new-example-cnf.md:49]: done

**Review by agentpoyo — APPROVED** [review]
LGTM

**Pull request merged**